src/utils/wishlist-util.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+
+async function addToWishlistUtil(dispatch, product, login) {
+  try {
+    const {
+      data: { wishlist },
+    } = await axios.post(
+      "/api/user/wishlist",
+      {
+        product,
+      },
+      {
+        headers: {
+          authorization: login,
+        },
+      }
+    );
+
+    dispatch({ type: "ADD_TO_WISHLIST", payload: wishlist });
```

Comment:
Suggestion- It's better to declare constants for dispatch types, instead of using strings as it will be least error prone.